### ISSUE TITLE
make command fails when installing onedrive to LM21.1

### ISSUE THREAD

**Issue by mikespolk** [skilion/onedrive#550]
I have formerly succeeded in installing onedrive sync client but now the make command seems to fail.
Here is scrap of the last command lines and responses:

lm@lm1-pc:~/temp$ **sudo apt install --no-install-recommends --no-install-suggests onedrive**
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  libllvm11 libphobos2-ldc-shared98
The following NEW packages will be installed:
  libllvm11 libphobos2-ldc-shared98 onedrive
0 upgraded, 3 newly installed, 0 to remove and 0 not upgraded.
Need to get 21,8 MB of archives.
After this operation, 92,8 MB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libllvm11 amd64 1:11.1.0-6 [19,6 MB]
Get:2 https://download.opensuse.org/repositories/home:/npreining:/debian-ubuntu-onedrive/xUbuntu_22.04 ./ onedrive 2.4.25-1+np1 [749 kB]
Get:3 http://archive.ubuntu.com/ubuntu jammy/universe amd64 libphobos2-ldc-shared98 amd64 1:1.28.0-1ubuntu1 [1 487 kB]
Fetched 21,8 MB in 4s (5 257 kB/s)                     
Selecting previously unselected package libllvm11:amd64.
(Reading database ... 577662 files and directories currently installed.)
Preparing to unpack .../libllvm11_1%3a11.1.0-6_amd64.deb ...
Unpacking libllvm11:amd64 (1:11.1.0-6) ...
Selecting previously unselected package libphobos2-ldc-shared98:amd64.
Preparing to unpack .../libphobos2-ldc-shared98_1%3a1.28.0-1ubuntu1_amd64.deb ...
Unpacking libphobos2-ldc-shared98:amd64 (1:1.28.0-1ubuntu1) ...
Selecting previously unselected package onedrive.
Preparing to unpack .../onedrive_2.4.25-1+np1_amd64.deb ...
Unpacking onedrive (2.4.25-1+np1) ...
Setting up libllvm11:amd64 (1:11.1.0-6) ...
Setting up libphobos2-ldc-shared98:amd64 (1:1.28.0-1ubuntu1) ...
Setting up onedrive (2.4.25-1+np1) ...
Processing triggers for man-db (2.10.2-1) ...
Processing triggers for libc-bin (2.35-0ubuntu3.1) ...

lm@lm1-pc:~/temp$ **sudo apt install git libcurl4-openssl-dev libsqlite3-dev**
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
git is already the newest version (1:2.34.1-1ubuntu1.10).
libcurl4-openssl-dev is already the newest version (7.81.0-1ubuntu1.13).
libsqlite3-dev is already the newest version (3.37.2-2ubuntu0.1).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

lm@lm1-pc:~/temp$ **git clone https://github.com/skilion/onedrive.git**
Cloning into 'onedrive'...
remote: Enumerating objects: 1123, done.
remote: Counting objects: 100% (7/7), done.
remote: Compressing objects: 100% (7/7), done.
remote: Total 1123 (delta 1), reused 2 (delta 0), pack-reused 1116
Receiving objects: 100% (1123/1123), 310.79 KiB | 1.48 MiB/s, done.
Resolving deltas: 100% (768/768), done.

lm@lm1-pc:~/temp$ **cd onedrive/**

lm@lm1-pc:~/temp/onedrive$ **make**
echo v1.1.4-2-g15b4730 >version
dmd -g -ofonedrive -O -L-lcurl -L-lsqlite3 -L-ldl -J. src/config.d src/itemdb.d src/log.d src/main.d src/monitor.d src/onedrive.d src/qxor.d src/selective.d src/sqlite.d src/sync.d src/upload.d src/util.d
make: dmd: No such file or directory
make: *** [Makefile:32: onedrive] Error 127
lm@lm1-pc:~/temp/onedrive$

Reason to have the ~/temp directory as default dir was because I wanted to make some kind of doublecheck, after getting similar error in the homedir.

How to get the make command working?

**Comment by abraunegg** [comment]
@mikespolk 
FYI this GitHub repository does not receive support. Please read: https://github.com/skilion/onedrive/issues/547

In regards to your error:
* you successfully installed the client that I maintain, following the correct installation instructions to use a package for Linux Mint
* you then are trying to manually build the 'skilion' client .. unsure why you are trying to do this when you have the correct client installed via the OpenSuSE repositories. Your error means you have not installed all the correct dependancies and/or activated the compiler so that you can compile.

**Comment by mikespolk** [comment]
Sorry, my bad,
I think I was just confused with some previous instructions and thus I ended up to combine instructions from https://github.com/skilion/onedrive (already retired if I understood correctly) and instructions from abraunegg github repo. Finally I succeeded to install Onedrive by simply following instructions here: https://github.com/abraunegg/onedrive/blob/master/docs/INSTALL.md

After that, just making first run and MS authentication by running "onedrive" and then enabling and starting it as service with commands
systemctl --user enable onedrive
systemctl --user start onedrive
I got it working and am happy now.

Thank you for your quick help!
